packages/salesforcedx-utils-vscode/src/cli/deployResultParser.ts

```diff
@@ -6,65 +6,71 @@
  */
 import { EOL } from 'os';
 
-export interface DeployError {
+export const CONFLICT_ERROR_NAME = 'sourceConflictDetected';
+
+export interface DeployResult {
   columnNumber?: string;
-  error: string;
+  error?: string;
   filePath: string;
   fullName?: string;
   lineNumber?: string;
+  state?: string;
   type: string;
 }
 
-export interface DeploySuccess {
-  state: string;
-  fullName: string;
-  type: string;
-  filePath: string;
-}
-
-export interface ForceSourceDeployErrorResult {
+export interface ForceSourceDeployErrorResponse {
```

Comment:
DeployResult and ForceSourceDeployErrorResult were confusing to me. Changed it to ForceSourceDeployErrorResponse to more accurately reflect what it represents. Same for the success response